.env_SAMPLE

```diff
@@ -197,3 +197,10 @@ fi
 
 # Enable the debug toolbar.
 # export ENABLE_DEBUG_TOOLBAR=True
+
+# Set an interal IP for the debug toolbar
+# This is necessary for it to work with docker
```

Comment:
```suggestion
# This is necessary for it to work with our Docker dev setup.
```

Reply:
No, I didn't mean that second thing. You're probably right that it's implied enough, at least until such potential time as we use Docker in production.

Reply:
Revised the suggested edit.